src/util/ValidationUtil.js

```diff
@@ -33,6 +33,17 @@ define(['okta'], function (Okta) {
     }
   };
 
+  // Validate that the password enroll matches for IdX. backwards compat, simplify later
+  fn.validatePasswordsMatchIdx = function (model) {
```

Comment:
why are we creating another method `validatePasswordsMatchIdx`? Can we just update `validatePasswordMatch `. Also I would avoid having idx in the function name.

Reply:
Ah, i did not realize we are using a shared Util file here. If you want you can keep that method but just rename it to avoid IDX.

Reply:
looks like it's only used in one place, isn't worth be a method in the util? or are we anticipating there will be more use cases?

Reply:
yeah, i agree, just moving this into the password enroll view